src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -4,21 +4,41 @@
  */
 
 import { User, WorkPackageStage } from 'shared';
+import { Dispatch, SetStateAction } from 'react';
 import { fullNamePipe } from '../../../utils/pipes';
 import PageBlock from '../../../layouts/PageBlock';
 import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material';
 import ReactHookTextField from '../../../components/ReactHookTextField';
-import { Control, Controller, FieldErrorsImpl } from 'react-hook-form';
+import { Controller } from 'react-hook-form';
 import { DatePicker } from '@mui/x-date-pickers';
-import { WorkPackageEditFormPayload } from './WorkPackageEditContainer';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 
 interface Props {
-  users: User[];
-  control: Control<WorkPackageEditFormPayload>;
-  errors: Partial<FieldErrorsImpl<WorkPackageEditFormPayload>>;
+  lead: string | undefined;
+  manager: string | undefined;
+  setManager: Dispatch<SetStateAction<string | undefined>>;
```

Comment:
also can you actually type these like this: (manager: string | undefined) => void, it means the same thing, it's just easier to understand what's going on at a glance this way